content/applications/hr/employees/new_employee.rst

```diff
@@ -0,0 +1,260 @@
+   :align: center
+   :alt: Create a new employee card.
+
+.. note::
+   The current company phone number and name is populated in the :guilabel:`Work Phone` and
+   :guilabel:`Company` fields.
+
+General information
+===================
+
+Required fields
+---------------
+
+- :guilabel:`Name`: Enter the employee's name.
+- :guilabel:`Company`: Select the :guilabel:`Company` from the drop-down menu that the new employee
+  is hired by, or create a new company by typing the name in the field.
+- :guilabel:`Working Hours`: In the :guilabel:`Work Information` tab, select the desired amount of
+  :guilabel:`Working Hours` from the drop-down menu.
+
+.. image:: new_employee/working-hours.png
+   :align: center
+   :alt: Working Hours are located in the Work Information tab.
+
+.. note::
+   :guilabel:`Working Hours` are based on a company's working time. Each individual working time is
+   specific to a company, and can be added and/or modified. To add or modify a working time, go to
+   the :menuselection:`Payroll app --> Configuration --> Working Times`, and add a new working time
+   or edit an existing one.
+
+Optional fields
+---------------
+
+- :guilabel:`Photo`: In the top right image box of the employee card, click on the :guilabel:`✏️
+  (pencil)` edit icon to select a photo to upload.
+- :guilabel:`Job Position`: Enter the employee's job position title.
+- :guilabel:`Tags`: Click on a :guilabel:`Tag` in the drop-down menu to add any tags applicable to
+  the employee. Any tag can be created in this field by typing it in. Once created, the new tag is
+  available for all employee cards. There is no limit to the amount of tags that can be added.
+- :guilabel:`Work Contact Information`: Enter the employees :guilabel:`Work Mobile`, :guilabel:`Work
+  Phone`, :guilabel:`Work Email`, and/or :guilabel:`Company` name. The :guilabel:`External Link`
+  button (next to :guilabel:`Company`) opens the company's :guilabel:`General Information` card.
+  Modifications can be made to this card. Click :guilabel:`Save` after making edits.
+- :guilabel:`Department`: Select the employee's department from the drop-down menu. The
+  :guilabel:`External Link` button opens the selected department's details. Modifications to the
+  :guilabel:`Department Name`, :guilabel:`Parent Department`, :guilabel:`Manager`, and
+  :guilabel:`Company` can be made here. Click :guilabel:`Save` after making edits.
+- :guilabel:`Manager`: Select the employee's :guilabel:`Manager` from the drop-down menu. The
+  :guilabel:`External Link` button opens the selected manager's :guilabel:`Employee Form`.
+  Modifications to the form can be made here. Click :guilabel:`Save` after any edits are made.
+- :guilabel:`Coach`: Select the employee's :guilabel:`Coach` from the drop-down menu. The external
+  link button opens the selected coach's :guilabel:`Employee Form`. Modifications to the form can be
+  made here. Click :guilabel:`Save` after any edits are made.
```

Comment:
That one was a gamble! I wasn't sure if I should use the GUI since they are elements IN the UI, and you're entering the info FOR the UI element- but rereading it now I see your point. Thank you!